pkgs/top-level/all-packages.nix

```diff
@@ -15645,6 +15645,13 @@ with pkgs;
     vm = callPackage ../applications/audio/open-music-kontrollers/vm.nix { };
   };
 
+  inherit (callPackages ../data/fonts/open-relay { })
+    constructium
+    fairfax
+    fairfax-hd
+    kreative-square
```

Comment:
Nit: Would it make more sense to use something like `open-relay-fonts.kreative-square` for the name? My Nix is rusty and to me this just looks like we're expecting users to install the fonts only by their name. But these fonts all come from `open-relay` so maybe it makes sense to group them together somehow?

Reply:
> If I were a Nixpkgs maintainer, I would personally like to see fonts in its own attrset like [...]

You /are/ a nixpkgs maintainer. You might not have the commit bit yet, but that's not a requirement to propose refactors in nixpkgs. It's more of a Do-ocracy, there's not a font institution inside the community responsible for a fonts attribute :laughing: 

If you're serious about this, open an issue, highlighting people previously involved. Maybe prototype on the "how" somewhere local before, and open a draft PR once it cooked a bit more. This shouldn't be self-merged anyways, but needs to be reviewed by people frequently interacting with the font system, so commit bits are orthogonal.